src/routes/CommunitySettings/MembershipRequestsTab/MembershipRequestsTab.store.js

```diff
@@ -0,0 +1,80 @@
+import {
+  FETCH_JOIN_REQUESTS,
+  FETCH_JOIN_REQUESTS_PENDING,
+  ACCEPT_JOIN_REQUEST,
+  DECLINE_JOIN_REQUEST
+} from 'store/constants'
+import fetchJoinRequestsQuery from 'graphql/queries/fetchJoinRequestsQuery'
+export const MODULE_NAME = 'MembershipRequestsTab'
+
+const defaultState = []
+
+export default function reducer (state = defaultState, action) {
+  const { error, type, payload } = action
+  if (error) return state
+
+  switch (type) {
+    case FETCH_JOIN_REQUESTS_PENDING:
+      return null
```

Comment:
shouldnt this return state?

Reply:
or you want it to blank everything our while its loading?

Reply:
Yes want it to blank everything rather than flash the axolotl while its loading